resources/assets/js/pages/page-form.js

```diff
@@ -220,6 +220,124 @@ function codePlugin() {
 }
 codePlugin();
 
+function drawIoPlugin() {
+
+    const drawIoUrl = 'https://www.draw.io/?embed=1&ui=atlas&spin=1&proto=json';
```

Comment:
This should be user configurable in case a self-hosted version is preferred.

Reply:
At first it will not be as I don't really know how/if self-hosted instances provide the same level & ease of integration via a public url such as this. I doubt they would otherwise it could be used by anybody.

Reply:
The default self-hosted instance is a static one (check https://github.com/jgraph/drawio#running) which does not provide any integration (Google Drive, Dropbox, …). Also you must specify `https=1` in query string if your own instance does not support TLS.

This lack of integration is not really important if files are saved as attachments inside Bookstack.